ireporter/settings.py

```diff
@@ -73,12 +73,37 @@
 # Database
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+if os.environ.get("APP_ENV") == "Development":
+    DATABASES = {
```

Comment:
Having the same code for Development and Production will raise a duplication issue with Maintainability which will give us a rating of C